prov/gni/src/gnix_vc.c

```diff
@@ -69,6 +70,367 @@ static int __gnix_vc_conn_ack_comp_fn(void *data);
 /*******************************************************************************
  * Helper functions
  ******************************************************************************/
+/**
+ * Insert vc based on AV type and favor FI_AV_TABLE
+ *
+ * @assumption: ep, ep->av, index, key are all valid non-null ptrs.
+ */
+static inline int __gnix_vc_insert(struct gnix_fid_ep *ep,
+				   fi_addr_t *index,
+				   gnix_ht_key_t *key,
+				   struct gnix_vc *vc)
+{
+	GNIX_TRACE(FI_LOG_EP_CTRL, "\n");
+	int ret, av_type = ep->av->type;
+
```

Comment:
My mistake.
> On Mar 28, 2016, at 8:59 AM, Howard Pritchard <notifications@github.com> wrote:
> 
> In prov/gni/src/gnix_vc.c <https://github.com/ofi-cray/libfabric-cray/pull/736#discussion_r57580597>:
> 
> > @@ -69,6 +70,367 @@ static int __gnix_vc_conn_ack_comp_fn(void *data);
> >  /*******************************************************************************
> >   * Helper functions
> >   ******************************************************************************/
> > +/**
> > + * Insert vc based on AV type and favor FI_AV_TABLE
> > + *
> > + * @assumption: ep, ep->av, index, key are all valid non-null ptrs.
> > + */
> > +static inline int __gnix_vc_insert(struct gnix_fid_ep *ep,
> > +				   fi_addr_t *index,
> > +				   gnix_ht_key_t *key,
> > +				   struct gnix_vc *vc)
> > +{
> > +	GNIX_TRACE(FI_LOG_EP_CTRL, "\n");
> > +	int ret, av_type = ep->av->type;
> > +
> maybe av->type should be enum fi_av_type?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/ofi-cray/libfabric-cray/pull/736/files/408e2a2ddff4bc9751170dc9ac6f6a66a970679b#r57580597>